src/c-writer.cc

```diff
@@ -1427,19 +1427,20 @@ void CWriter::WriteDataInitializers() {
   }
 
   for (const DataSegment* data_segment : module_->data_segments) {
-    if (data_segment->data.size()) {
-      Write(Newline(), "static const u8 data_segment_data_",
-            GlobalName(data_segment->name), "[] = ", OpenBrace());
-      size_t i = 0;
-      for (uint8_t x : data_segment->data) {
-        Writef("0x%02x, ", x);
-        if ((++i % 12) == 0)
-          Write(Newline());
-      }
-      if (i > 0)
+    if (data_segment->data.empty()) {
```

Comment:
Maybe keep this `if (!data_segment->data.empty())` to preserve the current control flow? The `if (empty) { continue; }` pattern seems more error-prone long-term.

Reply:
I could go either way here. I tend to prefer early-return/early-continue over indenting an entire block,  but also this part of the change seems different to the rest.

Reply:
In this file at least we seem to be using single line block without curly braces.  We are not consistent about in in wabt right now, but at least in this file it seems very common.   (IIUC the google/chromium style guide allows both).

Reply:
Sounds good -- it doesn't seem worth holding this PR up for style issues if we don't have strong (or the same) thinking on the desired style anyway. Will plan to merge now...